### ISSUE TITLE
latest preview version fails to track read chapters

### ISSUE THREAD

**Issue by jxsl13** [tachiyomiorg/tachiyomi#7267]
The latest one to three preview versions stopped to properly track already read chapters as well as jumping to the correct page of the last read chapter.

Also in the "New" tab one has to manually update the view before already new chapters are marked as read (greyed out)

**Comment by AntsyLich** [comment]
TL:DR If you want a stable experience, please use the Stable release.

Tachiyomi is in the middle of modernizing the codebase for the application. This is not all done at once but incrementally as Tachiyomi is a very big application. This means some things might not behave as normal, and if you notice anything that doesn't behave like normal it's most likely known by the developer and contributors as some things need to be intentionally broken during the incremental modernization. You can either strap in for the ride by still using Preview or not by using Stable. But do be warned that things might be weird and go up in flames. And remember to always have a backup handy.

To be more technical we are changing the UI framework to Compose which is a new and easier way to create UI. With this change to Compose the UI might change a tiny bit just as it is easier to work with. But it doesn't stop there we are also changing the database framework to SQLDelight which help reduce boilerplate and adds type safety. Not a lot will change with it but there still might be a few changes here and there. But as the database part needs migrations things might not work as well with the parts that still uses the old database framework.

**Comment by Seishirou101** [comment]
probably the same root cause as #7222

**Comment by jxsl13** [comment]
I don't want a stable experience, I want to report a potential bug that might not be fixed for stable version users.

**Comment by jxsl13** [comment]
the issue mentioned above is my behavior, just that is also extends towards actual chapters in specific mangas as well as keeping track of pages within those chapters.
